pkg/azurecluster/mutate_create_test.go

```diff
@@ -133,7 +133,7 @@ func TestAzureClusterCreateMutate(t *testing.T) {
 
 			handler, err := NewWebhookHandler(WebhookHandlerConfig{
 				BaseDomain: "k8s.test.westeurope.azure.gigantic.io",
-				CtrlCache:  ctrlClient,
+				CtrlReader: ctrlClient,
```

Comment:
why is it sometimes called `CtrlReader` and sometimes `CtrlCache`?

Reply:
Yeah, I noticed the same, so I changed these to `CtrlReader` everywhere, or at least I thought I did :smile: